src/aleph/vm/conf.py

```diff
@@ -94,7 +83,7 @@ def obtain_dns_ips(dns_resolver: DnsResolver, network_interface: str) -> list[st
         # Use a try-except approach since resolvectl can be present but disabled and raise the following
         # "Failed to get global data: Unit dbus-org.freedesktop.resolve1.service not found."
         try:
-            return list(resolvectl_dns_servers_ipv4(interface=network_interface))
+            return list(resolvectl_dns_servers(interface=network_interface))
```

Comment:
Should we only change this behavior if IPv6 is available in the VMs ?

Referring to the docstring "VMs only support IPv4 networking for now, we must exclude IPv6 DNS from their config."

Reply:
not sure I understand the comment, we filter that in the  instance.py (see below) not at the obtain_dns_ip step